GSA_Adapter/AdapterActions/Execute.cs

```diff
@@ -33,83 +33,39 @@ public partial class GSAAdapter
         /**** IAdapter Interface                        ****/
         /***************************************************/
 
-        public override Output<object,bool> Execute(string command, Dictionary<string, object> parameters = null, ActionConfig actionConfig = null)
+        public override Output<object,bool> Execute(IExecuteCommand command, ActionConfig actionConfig = null)
```

Comment:
This as a start looks so much better! But would just dispatch the commands as dynamic instead. I.e. change this method to something like:

```
        public override Output<object, bool> Execute(IExecuteCommand command, ActionConfig actionConfig = null)
        {
            return ExecuteCommand(command as dynamic, actionConfig);
        }
```

and then have methods like:

```
        private Output<object, bool> ExecuteCommand(BH.oM.Adapter.Commands.Close command, ActionConfig actionConfig = null)
        {
            var output = new Output<object, bool>() { Item1 = null, Item2 = false };
            output.Item2 = Close();
            return output;
        }
```

Could even be discussed if it would be worth using the same pattern as for the requests, that is having the adapter making the dynamic dispatch for us!